Tests/id_set.json

```diff
@@ -15302,6 +15314,17 @@
                     "PrintContext"
                 ]
             }
+        }, 
+        {
+            "Alexa Test Playbook": {
+                "name": "Alexa Test Playbook", 
+                "implementing_scripts": [
+                    "DeleteContext"
+                ], 
+                "implementing_commands": [
+                    "awis-info"
```

Comment:
I have a feeling that you forgot to update the test to use the domain command

Reply:
Forgot to re-run update_id_set

Reply:
make sure you run it in the pre-commit